x-pack/test/api_integration/apis/security_solution/hosts.ts

```diff
@@ -119,68 +106,77 @@ export default function ({ getService }: FtrProviderContext) {
-            __typename: 'CloudInstance',
           },
           machine: {
             type: [],
-            __typename: 'CloudMachine',
           },
           provider: ['digitalocean'],
           region: ['sfo2'],
-          __typename: 'CloudFields',
         },
-        __typename: 'HostItem',
       };
-
-      return client
-        .query<GetHostOverviewQuery.Query>({
-          query: HostOverviewQuery,
-          variables: {
-            sourceId: 'default',
-            hostName: 'zeek-sensor-san-francisco',
-            timerange: {
-              interval: '12h',
-              to: TO,
-              from: FROM,
-            },
-            defaultIndex: ['auditbeat-*', 'filebeat-*', 'packetbeat-*', 'winlogbeat-*'],
-            docValueFields: [],
-            inspect: false,
+      const {
+        body: { hostDetails },
+      } = await supertest
+        .post('/internal/search/securitySolutionSearchStrategy/')
+        .set('kbn-xsrf', 'true')
+        .send({
+          factoryQueryType: HostsQueries.details,
+          hostName: 'zeek-sensor-san-francisco',
+          timerange: {
+            interval: '12h',
+            to: TO,
+            from: FROM,
           },
+          defaultIndex: ['auditbeat-*', 'filebeat-*', 'packetbeat-*', 'winlogbeat-*'],
+          docValueFields: [],
+          inspect: false,
         })
-        .then((resp) => {
-          const hosts = resp.data.source.HostOverview;
-          expect(hosts).to.eql(expectedHost);
-        });
+        .expect(200);
+
+      expect(hostDetails).to.eql(expectedHostDetails);
     });
 
-    it('Make sure that we get Last First Seen for a Host', () => {
-      return client
-        .query<GetHostFirstLastSeenQuery.Query>({
-          query: HostFirstLastSeenGqlQuery,
-          variables: {
-            sourceId: 'default',
-            hostName: 'zeek-sensor-san-francisco',
-            defaultIndex: ['auditbeat-*', 'filebeat-*', 'packetbeat-*', 'winlogbeat-*'],
-            docValueFields: [],
-          },
+    it('Make sure that we get Last First Seen for a Host', async () => {
+      const { body: firstLastSeenHost } = await supertest
+        .post('/internal/search/securitySolutionSearchStrategy/')
+        .set('kbn-xsrf', 'true')
+        .send({
+          factoryQueryType: HostsQueries.firstLastSeen,
+          defaultIndex: ['auditbeat-*', 'filebeat-*', 'packetbeat-*', 'winlogbeat-*'],
+          docValueFields: [],
+          hostName: 'zeek-sensor-san-francisco',
         })
-        .then((resp) => {
-          const firstLastSeenHost = resp.data.source.HostFirstLastSeen;
-          expect(firstLastSeenHost).to.eql({
-            __typename: 'FirstLastSeenHost',
-            firstSeen: '2019-02-19T19:36:23.561Z',
-            lastSeen: '2019-02-19T20:42:33.561Z',
-          });
-        });
+        .expect(200);
+      const expected = {
+        isPartial: false,
+        isRunning: false,
+        rawResponse: {
+          took: 2,
```

Comment:
This and other instances of `took` aren't stable values, so they will fail on CI randomly. 
```suggestion
          took: firstLastSeenHost.rawResponse.took,
```